packages/vkui/src/lib/utils.test.ts

```diff
@@ -0,0 +1,45 @@
+import { addClassNameToElement, removeClassNameFromElement } from './utils';
+
+describe('addClassNameToElement', () => {
+  test('adds className to element', () => {
+    const div = document.createElement('div');
+    addClassNameToElement(div, 'a-class');
+
+    expect(div.getAttribute('class')).toEqual('a-class');
+
+    // allows to add duplicated class name
+    addClassNameToElement(div, 'a-class');
+    expect(div.getAttribute('class')).toEqual('a-class a-class');
+
+    addClassNameToElement(div, 'b-class');
+    expect(div.getAttribute('class')).toEqual('a-class a-class b-class');
+  });
+});
+
+describe('removeClassNameFromElement', () => {
+  test('removes className from element', () => {
+    const div = document.createElement('div');
+    div.setAttribute('class', 'a-class');
+
+    removeClassNameFromElement(div, 'a-class');
+    expect(div.getAttribute('class')).toEqual('');
+
+    // allows to remove duplicated class name
+    div.setAttribute('class', 'a-class b-class a-class');
+
+    removeClassNameFromElement(div, 'a-class');
+    expect(div.getAttribute('class')).toEqual('b-class a-class');
+
+    // allows to remove duplicated class name
+    div.setAttribute('class', 'a-class b-class a-class');
+
+    removeClassNameFromElement(div, 'b-class');
+    expect(div.getAttribute('class')).toEqual('a-class a-class');
+
+    removeClassNameFromElement(div, 'a-class');
+    expect(div.getAttribute('class')).toEqual('a-class');
+
+    removeClassNameFromElement(div, 'a-class');
+    expect(div.getAttribute('class')).toEqual('');
+  });
```

Comment:
Не хватает проверки удаление несуществующего класса

```suggestion
  });
  
  test('removing a non-existing class', () => {
    div.setAttribute('class', 'a-class b-class a-class');

    removeClassNameFromElement(div, 'с-class');
    expect(div.getAttribute('class')).toEqual('a-class b-class a-class');
  });
```

Reply:
Точно 😱

Reply:
⚡ https://github.com/VKCOM/VKUI/pull/5633/commits/87a3e678847b5817dba987965f73bb4217788ed2